python/interpret-core/interpret/provider/visualize.py

```diff
@@ -37,17 +37,35 @@ def _lazy_initialize(self):
         self.in_cloud_env = is_cloud_env(detected_envs)
 
         # NOTE: This is tested manually per release. Ignoring for coverage.
-        if self.in_cloud_env:  # pragma: no cover
+        if self.in_cloud_env == ENV_DETECTED.CLOUD:  # pragma: no cover
             log.info("Detected cloud environment.")
             warn(
-                "Cloud environment detected ({}): viz integration is still experimental.".format(
+                "Cloud environment detected are ({}): viz integration is still experimental.".format(
                     detected_envs
                 )
             )
             self.provider = InlineProvider(detected_envs=detected_envs, js_url=JS_URL)
         elif "docker-dev-mode" in detected_envs:
             log.info("Operating in docker development mode.")
             self.provider = InlineProvider(detected_envs=detected_envs)
+
+        elif self.in_cloud_env == ENV_DETECTED.BOTH_CLOUD_AND_NON_CLOUD:
+            log.info("Detected both cloud and non cloud environment.")
+            val = input("Type 'C' if you want to choose Cloud environment or 'NC' for Non Cloud Environment :")
+            if val == 'C':
```

Comment:
Thanks @prateekiiest -- The only change I'd suggest is that instead of calling input(...) we choose the cloud option by default when both are present since input waits for a user response.